lib/events/dynamoevents/dynamoevents.go

```diff
@@ -106,8 +106,8 @@ type Config struct {
 	ReadCapacityUnits int64 `json:"read_capacity_units"`
 	// WriteCapacityUnits is Dynamodb write capacity units
 	WriteCapacityUnits int64 `json:"write_capacity_units"`
-	// RetentionPeriod is a default retention period for events
-	RetentionPeriod time.Duration
+	// RetentionPeriod is a default retention period for events in seconds.
+	RetentionPeriod int64 `json:"audit_retention_period"`
```

Comment:
Any way to keep this a duration? I find it easier to reason about than a plain integer.

Reply:
Changed it to a duration now